changelogs/2023-07-27_gitlab-group-access-tokens.mdx

```diff
@@ -0,0 +1,20 @@
+---
+_schema: default
+title: GitLab group access tokens
+type: minor
+---
+You can now authenticate with GitLab using a group access token, to give CloudCannon access to more finely scoped sets of resources.
+
+**Features:**
+
+* Support for GitLab group access tokens.
+
+**Fixes:**
+
+* Dependency updates to patch security vulnerabilities.
+* Fixed some errors that could occur when editing a Markdown file with no frontmatter.
```

Comment:
frontmatter -> front matter

Reply:
Actually seems the internet is split on this issue so not a huge deal

Reply:
We favor `front matter` in our docs